orbit/pkg/table/firmware_eficheck_integrity_check/firmware_eficheck_integrity_check_darwin.go

```diff
@@ -0,0 +1,77 @@
+//go:build darwin
+// +build darwin
+
+// Package firmware_integrity_check implements a table
+// to perform an integrity check for Legacy EFI.
+package firmware_eficheck_integity_check
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/osquery/osquery-go/plugin/table"
+	"github.com/rs/zerolog/log"
+	"golang.org/x/sys/unix"
+)
+
+// Columns is the schema of the table.
+func Columns() []table.ColumnDefinition {
+	return []table.ColumnDefinition{
+		table.TextColumn("chip"),
+		table.TextColumn("output"),
```

Comment:
Should we call this field something more meaningful?
Or is this a pre-existing convention to call this data 'output'?

Reply:
No convention, I just thought that given that this table is for executing a particular command, then output is the output of such command. (Documented in the `schema/table/firmware_eficheck_integrity_check.yml`.)